testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeTlsKerberosDelegation.java

```diff
@@ -78,6 +79,10 @@ public EnvMultinodeTlsKerberosDelegation(
     @SuppressWarnings("resource")
     public void extendEnvironment(Environment.Builder builder)
     {
+        builder.configureContainer(TESTS, container -> {
```

Comment:
Can we move this logic to `HadoopKerberos` - so we could have a same config across all services in a PTs.

Reply:
moved